workers/unity/Packages/.gitignore

```diff
@@ -0,0 +1,11 @@
+io.improbable.gdk.buildsystem
```

Comment:
Should these all end with a `/` since they are folders?
Missing the mobile sdk

Reply:
It doesn't work if they are folders. Symlinks are treated as files afaict